firmware/Cargo.toml

```diff
@@ -103,3 +126,8 @@ mpu6050-dmp = { git = "https://github.com/TheButlah/mpu6050-dmp-rs", rev = "7712
 
 defmt = { git = "https://github.com/TheButlah/defmt", rev = "b5d5e316ea56250dcb25b3549bcf4bfdb1687796" }
 defmt-rtt = { git = "https://github.com/TheButlah/defmt", rev = "b5d5e316ea56250dcb25b3549bcf4bfdb1687796" }
+
+# https://github.com/esp-rs/esp-wifi#important
```

Comment:
I'm ok with continuing to provide the readme link as well, but you'll need to pin the commit version